devtoolsNG/basic/index.html

```diff
@@ -1,9 +1,15 @@
+<style>
+  .class1 {

```

Comment:
all this should be doable with:

#log > div:nth-child(even) {
  background-color: lightgrey;
}